metadata.json

```diff
@@ -82,6 +82,13 @@
         "8"
       ]
     },
+    {
+      "operatingsystem": "Rocky",
+      "operatingsystemrelease": [
+        "7",
```

Comment:
There is no Rocky Linux 7

Reply:
Apologies, you are absolutely correct. This was supposed to be 8 and 9. Not sure what I was thinking there.